api/src/main/java/gov/va/vro/api/responses/BipFileUploadResponse.java

```diff
@@ -3,6 +3,7 @@
 import io.swagger.v3.oas.annotations.media.Schema;
 import lombok.Builder;
 import lombok.Getter;
+import lombok.Setter;
```

Comment:
This setter is not used

Reply:
Thanks for pointing out. I forget to clean this up. Will do it later.